src/util.c

```diff
@@ -0,0 +1,47 @@
+#include "jemalloc/internal/util.h"
```

Comment:
`util.c` is probably too generic. Maybe `conf_parser.c`?

For new source files added, you will need to update the msvc projects as well, e.g. in
`./msvc/projects/vc2015/jemalloc/jemalloc.vcxproj`

Reply:
Its function prototype is defined in util.h, I think it makes sense to have its implementation in util.c? Or we want a separate `conf_parser.h`? That sounds too specific and this may be the only function in the file.

Reply:
I see. Yeah it looks fine in util.h. Let's keep it as is then.